### PR TITLE
8329322 : Convert PageFormat/Orient.java to use PassFailJFrame

### PR DESCRIPTION
**Pull request by Renjithkannath**
Hi Reviewers,

I have updated the test with PassFailJFrame along with printer availability check. Please review and let me know your suggestions.

Renjith.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8329322](https://bugs.openjdk.org/browse/JDK-8329322): Convert PageFormat/Orient.java to use PassFailJFrame (**Bug** - P4)

### Reviewers
 * [Abhishek Kumar](https://openjdk.org/census#abhiscxk) (@kumarabhi006 - Committer)
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18624/head:pull/18624` \
`$ git checkout pull/18624`

Update a local copy of the PR: \
`$ git checkout pull/18624` \
`$ git pull https://git.openjdk.org/jdk.git pull/18624/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18624`

View PR using the GUI difftool: \
`$ git pr show -t 18624`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18624.diff">https://git.openjdk.org/jdk/pull/18624.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18624#issuecomment-2037014752)

### CODE REVIEW

**Review by TejeshR13 — COMMENTED** [review]
Did you test it for other L&F also?

File: test/jdk/java/awt/print/PageFormat/Orient.java [hunk 1]
```diff
@@ -21,64 +21,42 @@
...
-         "On each page the oval will be wholly within the imageable area ",
-         "of the page. In a failing test the oval on the third page ",
-         "will be clipped against the imageable area.",
-         "Axes will indicate the direction of increasing X and Y"
-       };
-      Sysout.createDialog( );
-      Sysout.printInstructions( instructions );
-
+    private static final String INSTRUCTIONS =
+            "This test will automatically initiate a print.\n\n" +
```
Comment (kumarabhi006) [inline test/jdk/java/awt/print/PageFormat/Orient.java:47]: Can use Text Block style for better readability.

String INSTRUCTIONS = """
...""".

File: test/jdk/java/awt/print/PageFormat/Orient.java [hunk 2]
```diff
@@ -21,64 +21,42 @@
...
 
-/**
- * Orient.java
- *
- * summary:
+/*
+ * @test
+ * @bug 4236095
+ * @summary  Confirm that you get three pages of output, one
+ *           each in portrait, landscape, and reverse landscape
```
Comment (kumarabhi006) [inline test/jdk/java/awt/print/PageFormat/Orient.java:38]:
```suggestion
 *           each in portrait, landscape and reverse landscape
```

**Comment by kumarabhi006** [comment]
Is there any way we can test it with "print to pdf" option or physical printer is must?

**Comment by Renjithkannath** [comment]
Yes this will work on Print to pdf, I don't think physical printer is must.

**Comment by kumarabhi006** [comment]
> Yes this will work on Print to pdf, I don't think physical printer is must.

I am getting this dialog message "Printer status is not available at this time."

**Comment by Renjithkannath** [comment]
Its working for me, please make sure you have configured pdf printer as your default printer. In this test there is no option to select printer.

**Comment by kumarabhi006** [comment]
> Its working for me, please make sure you have configured pdf printer as your default printer. In this test there is no option to select printer.

Yeah... default was set to physical printer which is currently inaccessible. Working fine for me also.

**Review by kumarabhi006 — APPROVED** [review]
LGTM.

**Review by aivanov-jdk — APPROVED** [review]

**Pull request closed without merging**